website/src/layout.tsx

```diff
@@ -39,11 +58,17 @@ const Layout: React.FC = ({ children }) => {
                 Perseverance
               </span>
             </div>
+            <preferencesContext.Provider value={prefPack}>
```

Comment:
You definitely don't want to make multiple providers here, you should only have one. You can just wrap the whole return value of Layout in the provider.